maps/__Nadezhda/map/_Nadezhda_Colony_New.dmm

```diff
@@ -50722,6 +50728,8 @@
 	},
 /turf/simulated/floor/tiled/white/monofloor,
 /area/nadezhda/medical/surgery)
+"kRB" = (
```

Comment:
?

Reply:
theirs just a blank key?

Reply:
I don't know why it occurred, but I found and fixed it